src/gui/options_imp.cpp

```diff
@@ -28,38 +28,38 @@
  * Contact : chris@qbittorrent.org
  */
 
-#include <QTextStream>
-#include <QFileDialog>
-#include <QMessageBox>
-#include <QSystemTrayIcon>
+#include "options_imp.h"
+
+#include <cstdlib>
+
 #include <QApplication>
-#include <QDialogButtonBox>
 #include <QCloseEvent>
+#include <QDebug>
+#include <QDesktopServices>
 #include <QDesktopWidget>
+#include <QDialogButtonBox>
+#include <QFileDialog>
+#include <QMessageBox>
+#include <QSystemTrayIcon>
 #include <QTranslator>
-#include <QDesktopServices>
-#include <QDebug>
 
-#include <cstdlib>
+#ifndef QT_NO_OPENSSL
+#include <QSslCertificate>
+#include <QSslKey>
+#endif
 
-#include "app/application.h"
-#include "base/preferences.h"
-#include "base/utils/fs.h"
-#include "base/scanfoldersmodel.h"
 #include "base/bittorrent/session.h"
 #include "base/net/dnsupdater.h"
-#include "base/unicodestrings.h"
+#include "base/preferences.h"
+#include "base/scanfoldersmodel.h"
 #include "base/torrentfileguard.h"
+#include "base/unicodestrings.h"
+#include "base/utils/fs.h"
+#include "addnewtorrentdialog.h"
```

Comment:
The reason is not mixing Base and other application headers. It's not enough for you?
The second (IMO, not so good) way to do it is prepending all headers with its directories, but in that case headers affinity is not so clear (relative to current file).